src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -211,11 +211,11 @@ Item {
                  !statusBaseInput.multiline &&
                  edit.activeFocus
         anchors.right: parent.right
-        anchors.rightMargin: 11
+        anchors.rightMargin: 1
         anchors.verticalCenter: parent.verticalCenter
         type: StatusFlatRoundButton.Type.Secondary
-        width: 14
-        height: 14
+        width: 40
+        height: 40
```

Comment:
This works great! Can we make it a bit smaller though. With this size, the cursor is triggered way too soon.

Reply:
I suggest `25`